src/pbview.cpp

```diff
@@ -292,7 +292,7 @@ void pb_view::run_help()
 
 	code.append("}");
 
-	help_form.modify("helptext", "replace_inner", code);
+	help_form.modify("help_text", "replace_inner", code);
```

Comment:
I have no idea what is going on here, however I reckon this string should remain `helptext`.

Reply:
Yes definitely, I'm not sure when that got changed. I'll revert it.